DOCS-437: getMacro (#10614)
ClickHouse/ClickHouse: docs/en/sql-reference/functions/other-functions.md

Returns a string with the name of the host that this function was performed on. For distributed processing, this is the name of the remote server host, if the function is performed on a remote server.

## getMacro {#getmacro}

> Gets the parameter value from the [macros](../../operations/server-configuration-parameters/settings.md#macros) section of the server configuration file `config.xml`.

Reviewer: ```suggestion
Gets a named value from the [macros](../../operations/server-configuration-parameters/settings.md#macros) section of the server configuration.
```

**Syntax** 

```sql
getMacro(name);
```

**Parameters**

> - `name` — Name of the parameter from the `macros` section. [String](../../sql-reference/data-types/string.md#string).

Reviewer: ```suggestion
- `name` — Name to retrieve from the `macros` section. [String](../../sql-reference/data-types/string.md#string).
```

**Returned value**

> - Value of the specified parameter.

Reviewer: ```suggestion
- Value of the specified macro.
```

Type: [String](../../sql-reference/data-types/string.md).

**Example**

> 

Reviewer: First of all, we need to show an actual `macros` section configuration.

> Input table:

Reviewer: ```suggestion
An alternative way to get the same value:
```

```sql

> SELECT * FROM system.macros;

Reviewer: ```suggestion
SELECT * FROM system.macros
WHERE macro = 'test';
```
